data/migrations/V0052__copy_ofec_sched_a_aggregate_state_recipient.sql

```diff
@@ -0,0 +1,149 @@
+CREATE MATERIALIZED VIEW ofec_sched_a_aggregate_state_recipient_totals_2_mv AS
```

Comment:
Just out of curiosity, why the `2` in the name?  Are we still keeping the other materialized view?

Also, where is this materialized view being used?

Reply:
@ccostino thanks for pointing this out! I misunderstood this before - I'm going to push a new commit with the standard "create TMP/rename TMP" migration.